xwiki-platform-core/xwiki-platform-like/xwiki-platform-like-api/src/main/resources/templates/likers.vm

```diff
@@ -99,7 +99,8 @@
     #livetable_filterObfuscated($map)
     $jsontool.serialize($map)
 #else
-    <h1>$escapetool.xml($services.localization.render('like.likers.title', [$doc.plainTitle]))</h1>
+    #set($titleToDisplay = $services.localization.render('like.likers.title', ["<a href='$doc.getURL()'>$escapetool.xml($doc.plainTitle)</a>"]))
```

Comment:
To be tested but escaping html properties is probably safer.
```suggestion
    #set($titleToDisplay = $services.localization.render('like.likers.title', ["<a href='$escapetool.xml$doc.getURL())'>$escapetool.xml($doc.plainTitle)</a>"]))
```

Reply:
I'm pretty sure we shouldn't escape this URL: https://github.com/search?q=repo%3Axwiki%2Fxwiki-platform+%2Fhref%3D%27.*getURL%2F&type=code

Most of those are not escaped ...